install/v2plugin/Dockerfile

```diff
@@ -10,6 +10,12 @@ RUN mkdir -p /run/docker/plugins /etc/openvswitch /var/run/contiv/log \
  && wget https://github.com/andyshinn/alpine-pkg-glibc/releases/download/2.23-r1/glibc-2.23-r1.apk \
  && apk --no-cache add glibc-2.23-r1.apk
 
-COPY netplugin netmaster netctl startcontiv.sh /
+# copy in binaries and scripts
+ARG TAR_FILE
+ADD ${TAR_FILE} /
```

Comment:
If you're copying local files, use `COPY` instead of `ADD` ... generally you will not want to use `ADD` for anything, and all of Docker's official Dockerfiles have moved from `ADD` to `COPY` now

Reply:
the ADD exploded the tar file, I'd have to create another layer for a RUN to expand, there is nothing suggesting deprecation on the docs:
https://docs.docker.com/engine/reference/builder/#add

Reply:
Oh, right. "TAR_FILE" lol (used to looking for .tar.gz and so on) 😂